Lib/test/test_ssl.py

```diff
@@ -389,6 +389,7 @@ def test_random(self):
         ssl.RAND_add(bytearray(b"this is a random bytearray object"), 75.0)
 
     @unittest.skipUnless(os.name == 'posix', 'requires posix')
+    @unittest.skipUnless(hasattr(os, 'fork'), 'need os.fork')
```

Comment:
@unittest.skipUnless(os.name == 'posix', 'requires posix') was a dummy check for hasattr(os, 'fork'). You should remove it. Same remark for your other change.